### PR TITLE
New bridge setup

### PR DESCRIPTION
**Pull request by tutacrypto**

### CODE REVIEW

File: other/cross-chain/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+## Bridging between different networks
+
+Bridges are what allow agEUR and the ANGLE token to be available on different chains. At the top level, the bridges for agEUR (and ANGLE) all do the same thing: they lock agEUR (or ANGLE) in a smart contract on the origin network, and mint new tokens on the destination network. When bridging back agEUR (or ANGLE) from the destination chain, tokens are burnt there and released from the smart contract on the origin chain.
+
+AgEUR can be bridged between different networks using our new LayerZero infrastructure. It was previously relying on Multichain, which is still used for ANGLE. 
+
+{% page-ref page="layer-zero-setup.md" %}
+
+{% page-ref page="multichain-setup.md" %}
```
Comment (sogipec) [inline other/cross-chain/README.md:24]: Like we need this page on top that says for each chain which bridge solutions are supported for agEUR and ANGLE

File: other/cross-chain/layer-zero-setup.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+# 🌉 Angle LayerZero bridge infrastructure
+
+Angle is also based on a [LayerZero](https://layerzero.network/) infrastructure to bridge agEUR. LayerZero is a decentralized message passing solution made up of relayers sending messages between networks, and of contracts sending and receiving the messages and the tokens.
+
+At the top level, users will just have to specify an amount and a destination chain, and will be able to send agEUR from one chain to the other with one transaction. 
+
+![lz bridge infra user](../../.gitbook/assets/bridge-infra-user.png)
+
+## Bridging details
```
Comment (sogipec) [inline other/cross-chain/layer-zero-setup.md:13]: This is global and not only for Layer Zero

File: other/cross-chain/multichain-setup.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+description: Angle Multichain setup for bridging
+---
+
+# Angle Multichain setup
+
+The first bridging solution adopted by Angle to bridge tokens was Multichain. Here is the bridge setup for agEUR and ANGLE when using Multichain. 
+
+## Bridge Solutions
+
+| Chain         | Bridge                                                                 | agEUR  | ANGLE         |
```
Comment (sogipec) [inline other/cross-chain/multichain-setup.md:11]: I'd keep this table with for each token the list of supported tokens and potentially say whether native and direct wrapping is supported or not

Comment (tutacrypto) [reply other/cross-chain/multichain-setup.md:11]: What do you mean by this?

Comment (sogipec) [reply other/cross-chain/multichain-setup.md:11]: Like we still need an informative table that says for each chain which bridges are supported, and if people using these bridges will have to do the wrapping or not

Comment (sogipec) [reply other/cross-chain/multichain-setup.md:11]: For some bridges you'll still have to do it. We won't promote it, but it'll be a possibility. Like Multichain

File: SUMMARY.md [hunk 4]
```diff
@@ -27,93 +27,95 @@
...
+  - [📈 Interest](governance/veangle/interest.md)
+  - [🔓 Increasing your veANGLE balance](governance/veANGLE/increasing-veANGLE.md)
+- [💂 Angle Guardian](governance/guardian.md)
 
 ## Other Aspects
 
-* [⛓ Cross-Chain ANGLE and agEUR](other/cross-chain.md)
-* [⚙️ Algorithmic Market Operations](other/amo.md)
-* [⚡️ Flash Loans](other/flash-loans.md)
+- [⛓ Angle cross-chain](other/cross-chain/README.md)
```
Comment (sogipec) [inline SUMMARY.md:73]: Angle Cross-Chain Setup

**Pull request merged**